src/App.tsx

```diff
@@ -1,106 +1,40 @@
 import React from 'react';
-import {
-  BrowserRouter as Router,
-  Routes,
-  Route,
-  Navigate,
-} from 'react-router-dom';
 import './App.css';
-
-import Navbar from './components/Navbar';
-import AllUsers from './components/AllUsers';
-import Login from './components/Login';
-import TaskList from './components/TaskList';
-import Ticket from './components/Ticket';
-import AllProject from './components/AllProject';
-import Project from './components/Project';
-import AddUserX from './components/AddUserX';
+import jwt_decode from 'jwt-decode';
+import { useQuery, gql } from '@apollo/client';
+import AppRouter from './Router';
+import LoginContext from './context/LoginContext';
+import { GetOneUser } from './schemaTypes';
 
 function App(): JSX.Element {
-  interface IProtectedRoute {
-    user: string | null;
-    children: JSX.Element;
+  interface IdecodedToken {
```

Comment:
FYI, typo to be changed later into `IDecodedToken` with a capital `D`

Reply:
OK